tasks/main.yml

```diff
@@ -1,4 +1,16 @@
 ---
+# make sure we only accept Python version >=3
+# should use end_host here as this ends it for all hosts, but only available in Ansible >= 2.8
+- debug:
+    msg: >
+      "Please ensure that you use Python 3 to execute this Ansible playbook/role. "
+      "AiiDA requires Python 3. The simplest is to run the ansible command with "
+      "`-e 'ansible_python_interpreter=/usr/bin/python3'`, depending on where your "
+      "Python 3 interpreter is."
+  when: ansible_python_version is version('3', '<')
+- meta: end_play
+  when: ansible_python_version is version('3', '<')
```

Comment:
The issue we saw before was that this ends, but with "success" exit status on python 2.7.
Can this be changed so that this fails on python 2.7?

Reply:
Fixed